### PR TITLE
add metrics service config example

### PR DESCRIPTION
**Pull request by JaredTan95**
Signed-off-by: JaredTan95 <jian.tan@daocloud.io>

Commit Message: add metrics service config example
Additional Description:
Risk Level:
Testing:
Docs Changes: add metrics service config example

### CODE REVIEW

File: api/envoy/config/metrics/v3/metrics_service.proto [hunk 1]
```diff
@@ -21,6 +21,23 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Metrics Service is configured as a built-in *envoy.stat_sinks.metrics_service* :ref:`StatsSink
 // <envoy_v3_api_msg_config.metrics.v3.StatsSink>`. This opaque configuration will be used to create
 // Metrics Service.
+//
+// For example, with the following configurations (in YAML):
+//
+// MetricsService Config
+//
+// .. code::
```
Comment (dmitri-d) [inline api/envoy/config/metrics/v3/metrics_service.proto:29]: Please use `.. code-block:: yaml` here

File: api/envoy/config/metrics/v3/metrics_service.proto [hunk 2]
```diff
@@ -21,6 +21,23 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Metrics Service is configured as a built-in *envoy.stat_sinks.metrics_service* :ref:`StatsSink
 // <envoy_v3_api_msg_config.metrics.v3.StatsSink>`. This opaque configuration will be used to create
 // Metrics Service.
+//
+// For example, with the following configurations (in YAML):
```
Comment (dmitri-d) [inline api/envoy/config/metrics/v3/metrics_service.proto:25]: I think next three lines will sound a bit weird when rendered; perhaps just `Example:` here?

File: api/envoy/config/metrics/v3/metrics_service.proto [hunk 3]
```diff
@@ -21,6 +21,23 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
...
+//
+// .. code::
+//
+//   ...
+//  stats_sinks:
+//    - name: envoy.stat_sinks.metrics_service
+//      typed_config:
+//        "@type": type.googleapis.com/envoy.config.metrics.v3.MetricsServiceConfig
+//        transport_api_version: V3
+//        grpc_service:
```
Comment (dmitri-d) [inline api/envoy/config/metrics/v3/metrics_service.proto:37]: Would it make sense to include other parameters in the example too?

Comment (JaredTan95) [reply api/envoy/config/metrics/v3/metrics_service.proto:37]: Like `address`, `port` of `grpc_service `?

Comment (JaredTan95) [reply api/envoy/config/metrics/v3/metrics_service.proto]: What I want to show here is a complete example, But after thinking about it, I decided to remove the other parameters and keep only `metric service config`

**Comment by JaredTan95** [comment]
@phlax Hi, I am not very familiar with the format, could you give me a comment about how to fix CI filas?

**Comment by alyssawilk** [comment]
if you click through the failure you should be able to find logs.  alternately running the "fix_format" scripts via ci/do_ci.sh may help

**Comment by yanavlasov** [comment]
This is actually a proto format error. Did you run `./tools/proto_format/proto_format.sh fix`

**Comment by JaredTan95** [comment]
> This is actually a proto format error. Did you run `./tools/proto_format/proto_format.sh fix`

Thanks, I ran `./tools/code_format/check_format.py fix ` only, I will try it.

**Comment by JaredTan95** [comment]
api/envoy/

**Review by lizan — APPROVED** [review]

**Pull request merged**